src/joystick/hidapi/SDL_hidapijoystick.c

```diff
@@ -294,9 +297,13 @@ static SDL_GamepadType SDL_GetJoystickGameControllerProtocol(const char *name, U
             0x1532, // Razer
             0x20d6, // PowerA
             0x24c6, // PowerA
+            0x294b, // Snakebyte
```

Comment:
These vendors should be added to the other places we check for XBox One vendors. Search for 0x2e24 in src and android-project to find them.

Reply:
I think I got all of these.